### PR TITLE
Add events

### PR DESCRIPTION
**Pull request by cpt22**
Events were added to the API. Currently, the new, create, edit, update, show, delete actions function as expected. The valid fields are title, description, start_date, end_date, image, and visibility.

### CODE REVIEW

File: app/controllers/api/v1/application_controller.rb [hunk 1]
```diff
@@ -1,9 +1,18 @@
 module Api
   module V1
     class ApplicationController < ActionController::API
+      include Pundit
```
Comment (cpt22) [inline app/controllers/api/v1/application_controller.rb:4]: Ignore everything in this file

**Comment by EddieXu123** [comment]
Ummm I need to learn more about Ruby syntax before I approve this review ask someone with more experience pls :)

**Review by EddieXu123 — APPROVED** [review]
LGTM!

File: db/migrate/20210913154155_create_active_storage_tables.active_storage.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# This migration comes from active_storage (originally 20170806125915)
```
Comment (anika-washburn) [inline db/migrate/20210913154155_create_active_storage_tables.active_storage.rb:1]: What is this file for?

**Review by anika-washburn — APPROVED** [review]
Besides the one question, the changes look good! Don't understand every single thing, but for the most part everything makes sense.

**Pull request merged**